shared/chat/top-line.tsx

```diff
@@ -24,38 +25,42 @@ class FilteredTopLine extends PureComponent<Props> {
     return `${this.props.numSearchHits}`
   }
   render() {
-    const {participants, showBold, usernameColor} = this.props
-    const boldOverride = showBold ? globalStyles.fontBold : null
     return (
-      <Box
-        style={{
-          ...globalStyles.flexBoxRow,
-          alignItems: 'center',
-          flex: 1,
-          justifyContent: 'flex-start',
-          position: 'relative',
```

Comment:
Unsure what these styles were for but I checked desktop / ios and the inbox rows look ok.

Reply:
Yeah, I checked on both and couldn't see a reason for those styles either.